test/test.js

```diff
@@ -6,7 +6,7 @@ var assert = require('assert');
 var glob = require('glob');
 var SourceMapConsumer = require('source-map').SourceMapConsumer;
 var getLocation = require('./utils/getLocation.js');
-var buble = require('../dist/buble.umd.js');
+var buble = require('..');
```

Comment:
let's just use node's resolution algorithm (by doing so we kinda also test if the package is 'requirable' and if transpiled files are in the location specified by the `main` field)